src/transactionsOutcomeParsers/tokenManagementTransactionsOutcomeParser.spec.ts

```diff
@@ -0,0 +1,560 @@
+import { assert } from "chai";
+import { ErrParseTransactionOutcome } from "../errors";
+import { TokenManagementTransactionsOutcomeParser } from "./tokenManagementTransactionsOutcomeParser";
+import { SmartContractResult, TransactionEvent, TransactionLogs, TransactionOutcome } from "./resources";
+
+describe("test token management transactions outcome parser", () => {
+    const parser = new TokenManagementTransactionsOutcomeParser();
+
+    it("should test ensure error", () => {
+        const event = new TransactionEvent({
+            address: "erd1qqqqqqqqqqqqqqqpqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqzllls8a5w6u",
+            identifier: "signalError",
+            topics: ["Avk0jZ1kR+l9c76wQQoYcu4hvXPz+jxxTdqQeaCrbX8=", "dGlja2VyIG5hbWUgaXMgbm90IHZhbGlk"],
+            data: Buffer.from("QDc1NzM2NTcyMjA2NTcyNzI2Zjcy", "base64"),
+        });
+
+        const logs = new TransactionLogs({ events: [event] });
+        const scResult = new SmartContractResult({});
+        const txOutcome = new TransactionOutcome({ smartContractResults: [scResult], transactionLogs: logs });
+
+        assert.throws(
+            () => {
+                parser.parseIssueFungible(txOutcome);
+            },
+            ErrParseTransactionOutcome,
+            "encountered signalError: ticker name is not valid (user error)",
+        );
+    });
+
+    it("should test parse issue fungible", () => {
+        const identifer = "ZZZ-9ee87d";
+        const base64Identifier = Buffer.from(identifer).toString("base64");
+
+        const event = new TransactionEvent({
+            address: "erd18s6a06ktr2v6fgxv4ffhauxvptssnaqlds45qgsrucemlwc8rawq553rt2",
+            identifier: "issue",
+            topics: [base64Identifier, "U0VDT05E", "Wlpa", "RnVuZ2libGVFU0RU", "Ag=="],
+        });
+
+        const logs = new TransactionLogs({
+            address: "erd18s6a06ktr2v6fgxv4ffhauxvptssnaqlds45qgsrucemlwc8rawq553rt2",
+            events: [event],
+        });
+
+        const scResult = new SmartContractResult({});
```

Comment:
Also here (and below), if possible to not pass `scResult` (if not needed), let's do that. On the other hand, it's worth to emphasize that the parser is able to gather logs from the contract results, as well - and you have an example for that below :+1:

Reply:
Indeed. Changed!